//# Обработка Promise - catch
// Отработает ошибку от вышестоящих then. Если все в порядке, то catch вообще не выполнится. Но если любой из промисов будет отклонён (проблемы с сетью или некорректная json-строка, или что угодно другое), то ошибка будет перехвачена.

//# Синтаксис
promise.catch(function (error) {
  /**
   * обработает ошибку
   * .catch(f) это то же самое, что promise.then(null, f)
   */
});

//* function (error)
